content/events/EDCON.md

```diff
@@ -0,0 +1,21 @@
+---
+eventName: "EDCON"
+eventDescription: "A global non-profit conference committed to serving the #Ethereum ecosystem & enhancing Ethereum community interaction worldwide. "
+eventTopic: 
+  - "Web3"
+  - "Ethereum"
+  - "Blockchain"
+eventType: "Conference"
+
+location: "Bangkok, Thailand"
+venueName: "Tokyo, Japan"
```

Comment:
@laeccles Please reconfirm this since the location is in Thailand and the venue is on Japan.

Reply:
@WinsCodes I have updated the location